x-pack/legacy/plugins/maps/public/layers/styles/vector/components/color/dynamic_color_form.js

```diff
@@ -63,29 +90,33 @@ export function DynamicColorForm({
             defaultMessage: 'Custom color ramp',
           })}
           onChange={onColorMapSelect}
+          onColorMapTypeChange={onColorMapTypeChange}
           colorMapType={COLOR_MAP_TYPE.ORDINAL}
           color={styleOptions.color}
           customColorMap={styleOptions.customColorRamp}
           useCustomColorMap={_.get(styleOptions, 'useCustomColorRamp', false)}
           styleProperty={styleProperty}
+          showColorMapTypeToggle={showColorMapTypeToggle}
+        />
+      );
+    } else if (styleProperty.isCategorical()) {
```

Comment:
Why re-nest this?

Reply:
these are equally weighted code-paths, and helps readability at same indentations. early returns work better for guard statements on top of the function.